pkgs/servers/mpd/default.nix

```diff
@@ -105,6 +107,7 @@ in stdenv.mkDerivation rec {
       (mkFlag mmsSupport "mms")
       (mkFlag mpg123Support "mpg123")
       (mkFlag aacSupport "aac")
+      (mkFlag lameSupport "lame-encoder")
```

Comment:
This is in regard to all flags passed to ./configure: A lot of them are on "auto" mode, turning themselves on when the respective library is found. I don't know the rationale of passing these flags explicitly, so I'd remove those.